.github/workflows/push.yml

```diff
@@ -42,7 +42,9 @@ jobs:
           ./tools/dist
           ~/arduino_ide
         key: ${{ runner.os }}-${{ hashFiles('package/package_esp32_index.template.json',
+                                            'platform.txt',
                                             'tools/get.py',
+                                            'tools/platform.local.txt',                                            
```

Comment:
Is this one required?

Reply:
If this file changes, it means that cflags (any) have changed in CI building processs... really not sure if this is necessary.
Let me know.

Reply:
The cache includes the toolchains and the Arduino IDE.  The toolchains are unrelated to this.
As for the IDE, the question to ask is if `platform.local.txt` will introduce new options that affect it.  I think the answer is no, unless `platform.txt` does so and we override them with `platform.local.txt`. So I believe it's redundant.
That said, it doesn't really hurt to keep it, I'm just trying to understand if we really need it.  A concern would be if we keep customizing it and invalidating the cache everytime for each PR that does so.

Reply:
I believe that we will change it for finishing this PR and we will not change it for a long while after that.
There may be a very small chance of need for changing something in the `platform.local.txt` and not in the `platform.txt` in the future, but as I said, I think it will be very rare.

Reply:
`platform.txt` is mainly intended to the Arduino end user and as base for CI.
`tools/platform.local.txt` is only intended for CI, not used at all by the Arduino IDE.